### PR TITLE
Fix typos plot radiation tool

### PR DESCRIPTION
**Pull request by PrometheusPi**
This pull request fixes several typos in the `plotRadition` tool and adjust the code alignment. 
These issues were discovered by @ax3l in #1845 but this pull request was merged anyway.

### CODE REVIEW

File: src/tools/bin/plotRadiation [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 #
-# Copyright 2013-2017 Richard Pausch
```
Comment (ax3l) [inline src/tools/bin/plotRadiation]: @PrometheusPi uh oh :)

File: src/tools/bin/plotRadiation [hunk 2]
```diff
@@ -420,3 +477,5 @@ for myfile in args.path2Data:
     myfile.close()
 
     print(filename + "\t Done")
+
+#  LocalWords:  PIConGPU
```
Comment (ax3l) [inline src/tools/bin/plotRadiation]: what is that?

File: src/tools/bin/plotRadiation [hunk 3]
```diff
@@ -354,20 +405,26 @@ for myfile in args.path2Data:
...
-    # plot data: (only call!!!)
+    # plot data: (this is the only command that plots data)
     cax =  SP.pcolormesh(x, y, dataZoomed,
                          shading=my_interpolation,
                          cmap=my_cmap,
                          vmax=colorbarMax,
                          norm=colorNorm)
 
-    # set x and y range
+    # set x and y range (required after pcolormesh(?))
```
Comment (ax3l) [inline src/tools/bin/plotRadiation]: sure, you can always set your limits independent of your plot area.
in your case, the default might do (show all).

Comment (PrometheusPi) [inline src/tools/bin/plotRadiation]: pcolormesh seems to overwrite xlim in the case of logarithmic axis for matplotlib 1.3.x.
That's why I moved setting the axis limits after the pcolormesh call.

File: src/tools/bin/plotRadiation (hunk 2 above)
Comment (PrometheusPi) [inline src/tools/bin/plotRadiation]: This is for automated spell checks - ignore `PIConGPU` as typo.

File: src/tools/bin/plotRadiation (hunk 3 above)
Comment (ax3l) [inline src/tools/bin/plotRadiation]: they should be always behind the `plot()`, `imshow()` and `pcolormesh()` call afaik

File: src/tools/bin/plotRadiation (hunk 2 above)
Comment (ax3l) [inline src/tools/bin/plotRadiation]: in which editor? only emacs?

File: src/tools/bin/plotRadiation (hunk 3 above)
Comment (PrometheusPi) [inline src/tools/bin/plotRadiation]: Beforehand it worked even when `xlim`was set with the subplot. So I am not sure about that.

File: src/tools/bin/plotRadiation (hunk 2 above)
Comment (PrometheusPi) [inline src/tools/bin/plotRadiation]: actually not emacs - but anything that uses `aspell`

Comment (ax3l) [inline src/tools/bin/plotRadiation]: interesting but a bit random to put it in one file
one could think about a per-project or per-repository file but even there it's a bit... out of scope.

is it ok for you not to add it?

Comment (PrometheusPi) [inline src/tools/bin/plotRadiation]: Of course - I will remove it. (I can not use it with emacs anyway 😄 )

**Pull request merged**